src/lib/elf/elf.c

```diff
@@ -21,17 +21,40 @@ int elf_read_header(int fd, Elf32_Ehdr *header) {
 			header->e_ident[EI_MAG1] != ELFMAG1 ||
 			header->e_ident[EI_MAG2] != ELFMAG2 ||
 			header->e_ident[EI_MAG3] != ELFMAG3) {
-                return -EBADF;
+		return -EBADF;
 	}
 
 	return ENOERR;
 }
 
-int elf_read_interp(int fd, Elf32_Phdr *ph, char *interp) {
-	lseek(fd, ph->p_offset, SEEK_SET);
-	if (ph->p_filesz != read(fd, interp, ph->p_filesz)) {
+int elf_read_sh_table(int fd, Elf32_Ehdr *header, Elf32_Shdr *sh_table)
+{
```

Comment:
sorry, I should to grumble on codestyle again, we use "egyptian brackets" style for new scopes and functions, as specified in https://github.com/embox/embox/wiki/Code-Style#tabs-and-brackets
There are many occuriences of this issue in whole PR